service/controller/v1/resource/drainer/create.go

```diff
@@ -82,7 +83,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		p := []byte(UnschedulablePatch)
 
 		_, err := k8sClient.CoreV1().Nodes().Patch(n, t, p)
-		if apierrors.IsNotFound(err) {
+		if guest.IsAPINotAvailable(err) {
```

Comment:
Making this more graceful. Also backported to `node` resource for older clusters.